<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>change_worker</class>
 <widget class="QDialog" name="change_worker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Department name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="departmentName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Worker name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="workerLastName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>New name worker</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameWorkerNew"/>
        </item>
        <item>
         <widget class="QPushButton" name="changeName">
          <property name="text">
           <string>Change name</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>New worker position</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="positionWorkerNew"/>
        </item>
        <item>
         <widget class="QPushButton" name="changePosition">
          <property name="text">
           <string>Change position</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>New worekr salary</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="salaryWorkerNew"/>
        </item>
        <item>
         <widget class="QPushButton" name="ChangeSalary">
          <property name="text">
           <string>Change salary</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>Change all</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
